Notoday/HomePage.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  HomePage.swift
+//  Notoday
+//
+//  Created by Jamie Joung on 11/7/22.
+//
+
+import Foundation
+import SwiftUI
+import CoreData
+
+private func styleEmotions(_ value: String) -> Color {
```

Comment:
🚨 ⚠️ 🚨**DUPLICATE CODE ALERT** 🚨 ⚠️ 🚨

You should put this in an enum, as described elsewhere!

Also top-level functions and variables — meaning that they're defined outside a class — are generally discouraged. Like your `noteCreatedToday` var.